execution/src/posix/execution.cc

```diff
@@ -15,6 +15,11 @@
 #include <fcntl.h>
 #include <signal.h>
 
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+
+
```

Comment:
We shouldn't need these I think

Reply:
We do need them

Reply:
It looks to compile without, at lease locally on Linux,